client/src/components/TracerConcentration.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="md-layout md-gutter">
+    <div class="md-layout-item md-size-50">
+      <md-field>
+        <label>Tracer Concentration</label>
+        <md-select v-model="model.concentration_select">
+          <div v-for="(value, key) in selection_values" :key="key">
+            <md-option :value="key">{{value}}</md-option>
+          </div>
+        </md-select>
+      </md-field>
+    </div>
+    <div class="md-layout-item md-size-25">
+      <div
+        v-for="(value, key) in additional_values.fields"
+        :key="key"
+      >
+        <DoubleField
+          v-if="additionalFormControl(key)"
+          :param="key"
+          :value="value"
+          v-model="additional_values.fields[key]"
+        />
+      </div>
+      <md-field v-if="additionalFormControl('sample')">
+        <label>sample</label>
+        <md-select v-model="additional_values.selection.select">
+          <div v-for="(value, key) in additional_values.selection.values" :key="key">
+            <md-option :value="key">{{value}}</md-option>
+          </div>
+        </md-select>
+      </md-field>
```

Comment:
It would be great if this field was tied more explicitly to the model where the exception occurs. Its not really clear what additionalFormControl does or is expected to return here unless I go and look at the function.